tests/queries/0_stateless/00938_fix_rwlock_segfault_long.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: long, no-debug
+# Tags: long, no-debug, no-parallel
```

Comment:
Missing comments.

Reply:
Unfortunately, the explanation didn't fit.

Here, we query only the `name` column.
The exception should not happen. Let's fix the underlying cause.

Reply:
There is no reason the code in the stack trace should be called.